src/Instance/InstanceDetails/InstanceNotesView/InstanceNotesView.js

```diff
@@ -0,0 +1,67 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { orderBy } from 'lodash';
+
+import {
+  Accordion,
+} from '@folio/stripes/components';
+
+import {
+  getSortedNotes,
+} from '../../../utils';
+
+import InstanceNotesList from './InstanceNotesList';
+
+const InstanceNotesView = ({
+  id,
+  instance,
+  noteTypes,
+}) => {
+  const notesGroups = useMemo(() => {
+    if (!instance.notes?.length) {
```

Comment:
`isEmpty` from lodash
```suggestion
    if (isEmpty(instance.notes)) {
```